cli/azd/pkg/slice/slice.go

```diff
@@ -0,0 +1,12 @@
+package slice
+
+// Finds a value within a slice
+func Find[T any](slice []T, predicate func(value T) bool) *T {
```

Comment:
FWIW - https://pkg.go.dev/golang.org/x/exp/slices#IndexFunc.  I suspect at some point this package will be promoted into the stdlib.  Happy for you to keep what you have here, but wanted to make you aware that this existed (the go team published it with go 1.18 in order to get feedback on these sort of methods before adding them to stdlib and committing to them)

Reply:
Thanks for the info.  I will switch over to this :) Glad to see my implementation was pretty close to those of the stdlib authors.